test/e2e/testing-manifests/statefulset/mysql-upgrade/statefulset.yaml

```diff
@@ -9,47 +9,48 @@ spec:
     metadata:
       labels:
         app: mysql
-      annotations:
-        pod.beta.kubernetes.io/init-containers: '[
-          {
-            "name": "init-mysql",
-            "image": "mysql:5.7",
-            "command": ["bash", "-c", "
-              set -ex\n
-              [[ `hostname` =~ -([0-9]+)$ ]] || exit 1\n
-              ordinal=${BASH_REMATCH[1]}\n
-              echo [mysqld] > /mnt/conf.d/server-id.cnf\n
-              echo server-id=$((100 + $ordinal)) >> /mnt/conf.d/server-id.cnf\n
-              if [[ $ordinal -eq 0 ]]; then\n
-                cp /mnt/config-map/master.cnf /mnt/conf.d/\n
-              else\n
-                cp /mnt/config-map/slave.cnf /mnt/conf.d/\n
-              fi\n
-            "],
-            "volumeMounts": [
-              {"name": "conf", "mountPath": "/mnt/conf.d"},
-              {"name": "config-map", "mountPath": "/mnt/config-map"}
-            ]
-          },
-          {
-            "name": "clone-mysql",
-            "image": "gcr.io/google-samples/xtrabackup:1.0",
-            "command": ["bash", "-c", "
-              set -ex\n
-              [[ -d /var/lib/mysql/mysql ]] && exit 0\n
-              [[ `hostname` =~ -([0-9]+)$ ]] || exit 1\n
-              ordinal=${BASH_REMATCH[1]}\n
-              [[ $ordinal -eq 0 ]] && exit 0\n
-              ncat --recv-only mysql-$(($ordinal-1)).mysql 3307 | xbstream -x -C /var/lib/mysql\n
-              xtrabackup --prepare --target-dir=/var/lib/mysql\n
-            "],
-            "volumeMounts": [
-              {"name": "data", "mountPath": "/var/lib/mysql", "subPath": "mysql"},
-              {"name": "conf", "mountPath": "/etc/mysql/conf.d"}
-            ]
-          }
-        ]'
     spec:
+      initContainers:
```

Comment:
TBH, I didn't. I just converted it manually. Yeah it's not safe for not verifying it... :sweat_smile: